docs/documentation/graphing.asciidoc

```diff
@@ -32,6 +32,50 @@ define service {
 Often you can use the templates 'host-pnp' or 'srv-pnp' which set the right action_url automatically.
 =======
 
+==== Cookie Authentication
+
+If you want to use the cookie authentication for PNP4Nagios as well (for single sign-on) some additional changes need to be made. Modify every occurance of the following RewriteCond statement inside the cookie auth include file (usually located at '/usr/share/thruk/thruk_cookie_auth.include') from:
```

Comment:
The file '/usr/share/thruk/thruk_cookie_auth.include' would be overwritten on each update, maybe it would be better to copy it into /etc/apache2 or /etc/httpd and edit it there?